presto-docs/src/main/sphinx/release/release-0.221.rst

```diff
@@ -0,0 +1,54 @@
+=============
+Release 0.221
+=============
+
+General Changes
+---------------
+* Fix a null pointer exception during the stats collection phase in query planning.
+* Fix an issue where outer joins without equality predicates would sometimes run on a single node
+  when ``join_distribution_type`` was set to ``AUTOMATIC``.
+* Fix equality inference for ``VARCHAR`` columns.  This improves performance for queries that have
+  both constant predicates and join predicates on a ``VARCHAR`` column.
+* Add a variant of :func:`strpos` to return the position of the N-th instance of the substring.
+* Add :func:`strrpos` that returns the position of the N-th instance of a substring from the back of a string.
+* Add aggregation function :func:`entropy`.
+* Add classification aggregation functions :func:`classification_miss_rate`, :func:`classification_precision`,
+  :func:`classification_recall`, :func:`classification_thresholds`.
+* Add maximum standard error to :func:`approx_set`.
+* Add ``max_tasks_per_stage`` session property and ``stage.max-tasks-per-stage`` config property to
+  limit the number of tasks per stage for grouped execution.
+* Add encryption for spill files (see :doc:`/admin/spill`).
+
+Web UI Changes
+--------------
+* Add information about query warnings to the web UI.
+
+Raptor Changes
+--------------
+* Revert the change introduced in 0.219 to rebalance bucket assignment after restarting
+  the cluster. Automatic rebalancing can cause unexpected downtime when restarting the cluster
+  to resolve emergent issues.
+
+Hive Connector Changes
+----------------------
+* Decrease the amount of memory used by the coordinator when scheduling Hive splits.
+* Improve performance of writing large ORC files.
+
+SPI Changes
+-----------
+* Add ``PageSinkProperties`` for ``createPageSink`` in ``PageSinkProvider`` and
+  ``ConnectorPageSinkProvider``. It contains a boolean ``partitionCommitRequired``, which is
+  false by default.  See the note below about ``commitPartition`` for more information.
+* Add ``commitPartition`` to ``Metadata`` and ``ConnectorMetadata``. This SPI is coupled with
```

Comment:
IMO, in the future it would be great to add some of these details to the SPI overview in the documentation, and link to it.  https://prestodb.github.io/docs/current/develop/spi-overview.html